examples/module/src/module_arm/module_arm.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+import os
+from typing import Any, ClassVar, Dict, Mapping, Optional, Self, Tuple
+from viam.components.arm import Arm, JointPositions, KinematicsFileFormat, Pose
+from viam.operations import run_with_operation
+from viam.proto.app.robot import ComponentConfig
+from viam.proto.common import ResourceName
+from viam.resource.base import ResourceBase
+from viam.resource.types import Model, ModelFamily
+
+
+class ModuleArm(Arm):
+    # Subclass the Viam Arm component and implement the required functions
+    MODEL: ClassVar[Model] = Model(ModelFamily("rdk", "arm"), "module-arm")
+
+    def __init__(self, name: str):
+        # Starting position
+        self.position = Pose(
+            x=0,
+            y=0,
+            z=0,
+            o_x=0,
+            o_y=0,
+            o_z=1,
+            theta=0,
+        )
+
+        # Starting joint positions
+        self.joint_positions = JointPositions(values=[0, 0, 0, 0, 0, 0])
+        self.is_stopped = True
+        super().__init__(name)
+
+    @classmethod
+    def new(cls, config: ComponentConfig, dependencies: Mapping[ResourceName, ResourceBase]) -> Self:
+        arm = cls(config.name)
+        return arm
+
+    async def get_end_position(self, extra: Optional[Dict[str, Any]] = None, **kwargs) -> Pose:
+        return self.position
+
+    @run_with_operation
+    async def move_to_position(
+        self,
+        pose: Pose,
+        extra: Optional[Dict[str, Any]] = None,
+        **kwargs,
+    ):
+        operation = self.get_operation(kwargs)
+
+        self.is_stopped = False
+        self.position = pose
```

Comment:
Looks like if you're trying to mock this it should be set after the sleeps?

Reply:
This is true! I agree with this sentiment. I copied and pasted this from an example module arm, but will definitely make this change. Thanks for catching!